docs/en/latest/FAQ.md

```diff
@@ -626,6 +626,59 @@ This method only detects whether the APISIX data plane is alive or not. It does
 
 :::
 
+## What are the scenarios with high APISIX latency related to Etcd and how to fix them?
+
+Etcd is a component of APISIX service discovery and data storage, and its stability is related to the stability of APISIX.
```

Comment:
Currently APISIX don't have the ETCD service discovery module.

Reply:
> Thanks for the contribution!
> 
> Please wrap the version number with the back quota. Also, rename `Etcd` to `ETCD`.

OK, thanks for pointing it out.

Reply:
> Currently APISIX don't have the ETCD service discovery module.

However, as far as I understand it, the APISIX data plane uses HTTP-based long-polling to communicate with ETCD, which is a service discovery scenario (just not based on gRPC), isn't it?